src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -1752,11 +1787,43 @@ ValidateClauseList(List *clauseList)
 	{
 		Node *clause = (Node *) lfirst(clauseCell);
 
-		bool selectClause = IsSelectClause(clause);
-		bool joinClause = IsJoinClause(clause);
-		bool orClause = or_clause(clause);
+		/* produce a better error message for sublinks */
+		if (IsSublinkClause(clause))
+		{
+			ereport(ERROR, (errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+							errmsg("cannot perform distributed planning on this query"),
+							errdetail(
```

Comment:
We can fix this (citus_indent) side effect by keeping the line under 90 characters.